redis_cluster.c

```diff
@@ -1138,9 +1138,6 @@ PHP_METHOD(RedisCluster, keys) {
     } ZEND_HASH_FOREACH_END();
```

Comment:
@michael-grunder Why don't free data in previous line? It causes memory leak described in issue #1460

Reply:
It's probably an artifact of supporting both PHP 5 and php 7.